genestack_client/genestack_exceptions.py

```diff
@@ -1,4 +1,6 @@
 # -*- coding: utf-8 -*-
+import urllib2
```

Comment:
Maybe `from urllib2 import URLError`?  To indicate on the top of the module that only `URLError` is used.